docs/tutorial/vertex_cover.ipynb

```diff
@@ -0,0 +1,674 @@
+      "text/plain": [
+       "<Figure size 1620x700 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from qamomile.core.circuit.drawer import plot_quantum_circuit\n",
+    "\n",
+    "plot_quantum_circuit(qaoa_circuit)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "fmNHeUw7lXVI"
+   },
+   "source": [
+    "## Converting the Obtained QAOA Circuit and Hamiltonian for Qiskit\n",
+    "\n",
+    "We generate the QAOA circuit and Hamiltonian from the compiled instance using the `qamomile.qiskit.QiskitTranspiler` and `qamomile.qiskit.transpile_hamiltonian` converters. By utilizing these two methods, we can transform the QAOA circuit and Hamiltonian into a format compatible with Qiskit. This allows us to leverage Qiskit's quantum computing framework to execute and analyze."
```

Comment:
`transpile_hamiltonian` is the method of `QiskitTranspiler`, so this explanation is not correct.

Reply:
Fixed!